lib/migration.js

```diff
@@ -764,10 +764,19 @@ function mixinMigration(MySQL, mysql) {
     const colLength = columnMetadata && columnMetadata.dataLength || prop.length || prop.limit;
     const colPrecision = columnMetadata && columnMetadata.dataPrecision;
     const colScale = columnMetadata && columnMetadata.dataScale;
+    let enumList = '';
+    if (colType && colType === 'ENUM') {
+      if (prop.jsonSchema && prop.jsonSchema.enum) {
+        prop.jsonSchema.enum.forEach(item => {
+          enumList += `'${item}'`;
```

Comment:
do we need some separators like `,` to separate each enum value?

Reply:
@dhmlau. You are right. Thanks for pointing out. Fixed it.